packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -49,13 +48,13 @@ export const gcTombstoneGenerationOptionName = "gcTombstoneGeneration";
 export const gcDisableThrowOnTombstoneLoadOptionName = "gcDisableThrowOnTombstoneLoad";
 
 /**
- * This GC Option (on ContainerRuntime Options) allows an app to disable GC Sweep on old documents by incrementing this value.
+ * This undocumented GC Option (on ContainerRuntime Options) allows an app to disable GC Sweep on old documents
```

Comment:
I was thinking that instead of saying this can be used to disable sweep on old documents, this should say this allows configuring which documents can have sweep enabled. And then go on explaining how one can filter out old couments and why.

I feel that it changes the narrative a little - instead of saying disable sweep if there are issues, we are saying filter documents based on whatever logic you have.

Reply:
Good suggestion